modules/apps/commerce/commerce-payment-service/src/main/java/com/liferay/commerce/payment/internal/gateway/CommercePaymentGatewayImpl.java

```diff
@@ -50,10 +68,17 @@ public CommercePaymentEntry authorize(
 			throw new UnsupportedOperationException();
 		}
 
+		CommercePaymentIntegration commercePaymentIntegration =
+			_getCommercePaymentIntegration(commercePaymentEntry);
+
+		CommercePaymentEntry authorizeResult =
```

Comment:
Maybe we can name it this
```suggestion
		CommercePaymentEntry authorizedCommercePaymentEntry =
```